cooltools/lib/checks.py

```diff
@@ -297,13 +297,9 @@ def _is_compatible_trans_expected(
 
         # Check that view regions are named as in expected table.
         # Check region names:
-        _all_expected_regions = expected_df[["region1", "region2"]].values.flatten()
-        if not np.all(verify_view["name"].isin(_all_expected_regions)):
-            raise ValueError(
-                "View regions are not in the expected table. Provide expected table for the same regions"
-            )
+        if verify_cooler is not None:
```

Comment:
typo here - should be `verify_view` not `verify_cooler`

Reply:
thanks!